limits.c

```diff
@@ -27,10 +27,15 @@
 #include "motion_control.h"
 #include "limits.h"
 #include "report.h"
+#include "signals.h"
 #include "magazine.h"
+#include "nuts_bolts.h"
 
 #define HOMING_AXIS_SEARCH_SCALAR  1.1  // Axis search distance multiplier. Must be > 1.
-
+#define MAX_FORCE_SERVO_CYCLES 10
+#define MIN_FORCE_SERVO_CYCLES 3
+#define LONG_FORCE_SERVO_DELAY 1000
+#define SHORT_FORCE_SERVO_DELAY 200
```

Comment:
Trial and error. The delays compensate for settling time of the load cell and the max cycles ensure that the gripper doesn't oscillate indefinitely. Min cycles is there to make sure that the gripper tries at least twice at a slow rate after the second delay, since initially force servoing at a fast rate might cause overshoot.